datafusion/functions/src/math/monotonicity.rs

```diff
@@ -15,24 +15,17 @@
 // specific language governing permissions and limitations
 // under the License.
 
-use arrow::datatypes::DataType;
 use datafusion_common::{exec_err, Result, ScalarValue};
 use datafusion_expr::interval_arithmetic::Interval;
 use datafusion_expr::sort_properties::{ExprProperties, SortProperties};
 
-fn symmetric_unit_interval(data_type: &DataType) -> Result<Interval> {
```

Comment:
because this is also used in the bounds code, I added a constructor on `Interval`. I wasn't sure about adding others (e.g. `Interval::make_symmetric_pi_interval`) since right now they're only also used in the bounds code.

Reply:
It makes sense to me

Reply:
[78a943f](https://github.com/apache/datafusion/pull/11584/commits/78a943f38d90a7276ca9cdfef0f1d99cb91c4883)